app/views/pages/contact.html.erb

```diff
@@ -39,7 +39,7 @@
       To report a bug, please create <a href="https://github.com/forem/forem/issues/new/choose">a bug report</a> in our open source repository.
     </p>
     <p>
-      To request a feature, please visit <a href="https://forem.dev">forem.dev</a> and write a post!
+      To request a feature, please visit <a href="https://forem.dev">forem.dev</a> and Create Post!
```

Comment:
This should probably still be "write a post" 😅

Reply:
@Zhao-Andy 😅 you're right (or "write"?) let me fix this!